src/drawers/areaDrawer.ts

```diff
@@ -21,3 +22,34 @@ export class AreaSVGDrawer extends SVGDrawer {
     return super.getVisualPrimitiveAtIndex(0);
   }
 }
+
+export function makeAreaCanvasDrawStep(
+    d3AreaFactory: () => d3.Area<any>,
+    d3LineFactory: () => d3.Line<any>,
+): CanvasDrawStep {
+  return (context: CanvasRenderingContext2D, data: any[][], attrToAppliedProjector: AttributeToAppliedProjector) => {
+    const d3Area = d3AreaFactory();
+    context.save();
+    context.beginPath();
+    d3Area.context(context);
+    d3Area(data[0]);
+    context.lineJoin = "round";
+    const areaStyle = resolveAttributes(attrToAppliedProjector, ["fill", "opacity"], data[0], 0);
+    areaStyle['opacity'] = areaStyle['opacity'] ? areaStyle['opacity'] : 1;
+    areaStyle['opacity'] *= 0.25;
```

Comment:
so the area's opacity can be set externally to display selection. This mimics what CSS was doing because it was using the SVG fill-opacity attribute in plottable as well as CSS opacity from bp-chart to further reduce opacity when another series is selected.